ConditionsConsumed/runCMSDrivers_mc2018.sh

```diff
@@ -2,8 +2,7 @@
 echo "------------------------------"
 echo "Running step-1/9: GEN"
 echo "------------------------------"
-cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions 105X_upgrade2018_realistic_v6 -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
-
+cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions auto:run2_mc -n 5 --era Run2_2016 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
```

Comment:
I still think the Era here should be `--era Run2_2018`. Why are you setting it as `Run2_2016` ?

Reply:
My mistake, checking the source code:
 https://github.com/ravindkv/MyAlCaTools/blob/master/runCMSDrivers_mc2018.sh
it's possible to see that must be --era Run2_2018. I will fix that